### DISCUSSION TITLE
群晖Cloud-sync使用alist-docker部署WebDav对接夸克网盘可下载但无法上传

### DISCUSSION THREAD

**Discussion by quxiaorui** [AlistGo/alist#993] [General]
![alist设置](https://user-images.githubusercontent.com/17422635/164976931-24af3483-dd55-4f30-bdcd-dababe6ffa35.jpg)
docker日志显示upload error: Bad Parameter: [format_type is null!] ，请问是否还需要其他设置?
![docker日志](https://user-images.githubusercontent.com/17422635/164976928-a4967732-4070-41cf-a0bc-b7a2fb26dc43.jpg)

**Comment by quxiaorui** [comment]
前台网页上传正常，群晖的cloud-sync,hyper-backup都有些问题

**Reply by yuju520** [reply]
我也是这个问题，想要加密备份到网盘，但是显示权限不足。

**Comment by mwnu** [comment]
RaiDrive上传没有问题，File Station、Cloud Sync、Hyper Backup上传都有问题。
File Station可以用http连接上传，https没有权限。
Cloud Sync需要在网盘中删除@SynologyCloudSync文件夹（包括从回收站清空）之后重新创建连接可以同步，[详见](https://github.com/alist-org/alist/discussions/1068)。
Hyper Backup要么没有权限，要么目的地离线或目的地正忙。
由于日志没有足够的信息，只能挨个排查一遍，能试的都试了，排除反向代理、群晖系统设置、Alist配置、网盘来源的问题。
